api/src/util/pool.ts

```diff
@@ -0,0 +1,18 @@
+import config from './config';
+import logger from './logger';
+import pg from 'pg';
+
+/**
+ * Pool of Postgres connections to avoid overhead of connecting on every request.
+ */
+const pool: pg.Pool = new pg.Pool({
```

Comment:
Not really a comment, just wanted to know what the convention will be forward. Do we want to explicitly type the variables even in obvious cases?

Reply:
Umm idk. Probably not? I just did it here b/c it is an exported type - not really necessary though.